src/devices/Ws28xx/Ws28xx.cs

```diff
@@ -2,22 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Iot.Device.Graphics;
 using System.Device.Spi;
 
-namespace Iot.Device.Ws2812b
+namespace Iot.Device.Ws28xx
 {
-    public class Ws2812b
+    public class Ws28xx
     {
-        private readonly SpiDevice _spiDevice;
-        public BitmapImageNeo3 Image { get; }
+        protected readonly SpiDevice _spiDevice;
+        public BitmapImage Image { get; protected set; }
 
-        public Ws2812b(SpiDevice spiDevice, int width, int height = 1)
+        public Ws28xx(SpiDevice spiDevice, int width, int height = 1)
         {
             _spiDevice = spiDevice;
             _spiDevice.ConnectionSettings.ClockFrequency = 2_400_000;
```

Comment:
I know this isn't you but this seems incorrect that we override the SPI settings - must have missed it in the other review

Reply:
Fixed